python/pyspark/sql/tests/connect/test_connect_basic.py

```diff
@@ -25,6 +25,7 @@
 from pyspark.testing.utils import ReusedPySparkTestCase
 from pyspark.testing.connectutils import should_test_connect, connect_requirement_message
 from pyspark.testing.pandasutils import PandasOnSparkTestCase
+from pyspark.sql.connect.client import SparkConnectClientException
```

Comment:
@itholic mind reviewing this since you're working on error related improvement at https://github.com/apache/spark/pull/39137?

Reply:
Sure, let me take a look!